src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,24 @@
+/* eslint-disable implicit-arrow-linebreak */
+
 import React from 'react';
+import commentsFromServer from '../../api/comments';
+import { Comment } from '../../types/Comment';
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+type Props = {
+  postId: number,
+};
+
+export const CommentList: React.FC<Props> = ({ postId }) => {
+  const found = (element: Comment) => element.postId === postId;
```

Comment:
You are filtering through 'commentsFromServer' twice. Consider storing the filtered comments in a variable to improve performance 🚀